substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/Resources.java

```diff
@@ -56,6 +56,7 @@
 
     static class ResourcesSupport {
         final Map<String, List<byte[]>> resources = new HashMap<>();
+        final URLStreamHandler resourcesURLStreamHandler = createResourcesURLStreamHandler();
```

Comment:
I don't think that we need the `ResourceSupport.resourcesURLStreamHandler` field. The `Resources.createResourcesURLStreamHandler()` method could be moved to `JavaNetSubstitutions` instead.